torch/onnx/symbolic_opset9.py

```diff
@@ -1351,6 +1351,14 @@ def zeros_like(g, input, dtype=None, layout=None, device=None, pin_memory=False,
                 value_t=torch.tensor([0], dtype=sym_help.scalar_type_to_pytorch_type[dtype]))
 
 
+@parse_args('v', 'v', 'i', 'v', 'v', 'v')
+def new_zeros(g, self, sizes, dtype, layout, device, pin_memory=False):
+    if dtype is None:
+        dtype = self.type().scalarType()
```

Comment:
the `zeros` called below checks for cases where `dtype` is `None`. The check here would be duplicated though.

Reply:
But in general, it's not possible to not have the dtype, since new_zeros aten op gets the type from input tensor and sets dtype (that's the whole purpose of the op).